docs/app/Examples/views/Card/Variations/ColumnCount.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Card, Image } from 'stardust'
+
+const ColumnCountCards = () => (
+  <div>
+    <Card.Group itemsPerRow={6}>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card raised href='#column-count'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+    </Card.Group>
+  </div>
```

Comment:
Let's remove the extra `div` wrapper in all examples that can.  React only requires a single element to be returned, the type does not matter.  In this case, it will be a `Card.Group`.